RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.cc

```diff
@@ -114,6 +116,28 @@ PCCNTupler::PCCNTupler(edm::ParameterSet const& iConfig):
         pileUpToken=consumes<std::vector< PileupSummaryInfo> >(fPileUpInfoLabel);
         tree->Branch("nPU",&nPU,"nPU/I");
     }
+
+    if(includeJets){
+        hltjetsToken_=consumes<reco::CaloJetCollection >(edm::InputTag("ak4CaloJets"));
+        const int kMaxJetCal = 10000;
```

Comment:
10k ak4 jets can't fit into CMS acceptance.
This looks excessive regarding memory.